.github/workflows/run-tests.yml

```diff
@@ -6,7 +6,7 @@ jobs:
   test:
     runs-on: ${{ matrix.os }}
     strategy:
-      fail-fast: true
+      fail-fast: false
```

Comment:
Not a required change, but I favor all versions of the matrix always running to completion even if one fails.

Reply:
I agree with this. Thanks for pointing that out.